patterns/dynamic_values/config_modules/security_groups/security_groups.tf

```diff
@@ -151,11 +151,10 @@ output "f5_bastion" {
 output "bastion_vsi" {
   description = "Security Group for Bastion VSI"
   value = {
-    name           = "bastion-vsi-sg"
-    vpc_name       = var.bastion_vpc_name
-    resource_group = "${var.prefix}-${var.bastion_vpc_name}-rg"
-    rules          = var.bastion_vsi_rules
-    access_tags    = []
+    name        = "bastion-vsi-sg"
+    vpc_name    = var.bastion_vpc_name
+    rules       = var.bastion_vsi_rules
```

Comment:
I think it might be risky to remove resource_group here as it is used directly in the virtual_server.tf for the VSI pattern.

I think when deploying VSIs in existing VPCs, the resource group should be known (or asked for by consumer) and sent to the VSI submodule.

If you really don't care about this output, I suppose you could just pass in a known vpc_name to this input var for bastion so that it doesn't fail?